### PR TITLE
JavaScript With ES6 syntax

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
- In this project, I have refactored my Awesome books project to use ES6 syntax

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+const findHight = () => {
+  if (window.innerHeight < document.body.scrollHeight) {
+    document.querySelector('footer').style.position = 'relative';
+  } else {
+    document.querySelector('footer').style.position = 'fixed';
+  }
+};
```
Comment (mikethreels) [inline index.js:7]: - [ ] according to the requirements you should Create a directory called modules. and Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better! please do so before requesting another review.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  const suffix = (currentHour >= 12 && currentHour !== 0) ? 'pm' : 'am';
+  if (currentHour > 12) {
+    newHour = currentHour - 12;
+  } else if (currentHour === 0) {
+    newHour = 12;
+  } else {
+    newHour = currentHour;
+  }
+  return `${newHour}:${CurrentTime.getMinutes()}:${CurrentTime.getSeconds()} ${suffix}`;
+};
```
Comment (mikethreels) [inline index.js:97]: - [ ] The current date which is displayed below the navigation bar, for this requirement, you should use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time) for more details.

- Package is added as a .js file to the project (in full or minified version)
- Module is imported import { DateTime } from "luxon";

File: README.md [hunk 3]
```diff
@@ -1,31 +1,60 @@
...
-. Setup git clone https://github.com/SiansandaTamara/awsomebook cd awsome book Usage Open the index file of the project in the browser to see the project
+## Built With
+- HTML
+- CSS
+- JavaScript
+## Live Demo (if available)
 
-Authors 👤 Tamara Siansanda
- Authors👤 Cyril mucheml 
+[Live Demo Link](https://siansandatamara.github.io/framework-awesomebook/)
```
Comment (mikethreels) [inline README.md:17]: - [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Cyril, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @cyrilmuchemi,** 

### **Great work on making the changes 👏** 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## **Suggested changes**
_Check the comments under the review._

- **Hello mate as the previous tse said the requirements state that the files should be divided. please make sure to submit the PR after completing all requirements.**

Capture web_26-4-2022_125030_github com



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by keroloussamy — APPROVED** [review]
Hi @cyrilmuchemi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**